projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -1451,7 +1459,7 @@ export class IgxTimePickerComponent implements
      * @hidden
      */
     public hideOverlay(): void {
```

Comment:
close or closeDialog maybe

Reply:
We won't change this name for now as it will introduce breaking change and we don't have migration for method renaming. Writing a migration will take some time and we can do it later.